tests/testthat/test_util-RunWorkflow.R

```diff
@@ -20,11 +20,18 @@ lData <- gsm::UseClindata(
   )
 )
 
+# Quiet output
+quiet_RunWorkflow <- function(...) {
```

Comment:
we used to have a `bQuiet` parameter in these functions. should we go that route again, or is this wrapper function preferred? @jwildfire

Reply:
This was definitely aiming at "good enough for now." I needed to quiet down this test so I could more easily see what was actually failing. I *really* want to work on revising this set of tests to speed things up (#1635), and doing so in the context of any desired refactors (like perhaps adding this argument) would definitely make sense.

Reply:
Thinking it through... I know I find it annoying that {httr2} has a message about using the cache that can't (easily) be silenced, and thus fills up a log for something I run with it. I'd definitely appreciate being able to change the verbosity level. 

Per [Hadley's in-progress book](https://design.tidyverse.org/boolean-strategies.html) and [some discussions I've seen](https://fosstodon.org/@coolbutuseless/112742297912462306), it might be better to use  enumerated values rather than a boolean, in case we want to have multiple levels in the future. Something like `strVerbosity = c("verbose", "quiet")`

Reply:
@lauramaxwell  @jonthegeek As much as I loved having something called `bQuiet`, I definitely don't want to go down that route again (it was a fairly major pain to maintain). I'm fine with using `suppressMessages()` in testing as needed.